src/risclog/sqlalchemy/tests/fixtures/alembic/versions/3f735078d5a8_foo.py

```diff
@@ -5,19 +5,17 @@
 Create Date: 2013-08-12 10:59:01.394646
 
 """
+from alembic import op
+import sqlalchemy as sa
 
 # revision identifiers, used by Alembic.
 revision = '3f735078d5a8'
 down_revision = None
 
-from alembic import op
-import sqlalchemy as sa
-
```

Comment:
Did that happen automatically? It is different from the generated template, but does not matter so much.

Reply:
No, I had to do it myself to fix flake8 errors.